angular.json

```diff
@@ -94,7 +94,7 @@
             "main": "src/test.ts",
             "polyfills": "src/polyfills-test.ts",
             "tsConfig": "src/tsconfig.spec.json",
-            "karmaConfig": "src/karma.conf.js",
+            "karmaConfig": "tests/karma.conf.js",
```

Comment:
For neatness, I placed the Karma config under `tests` rather than the root directory.